pandas/tests/frame/test_apply.py

```diff
@@ -1357,3 +1357,10 @@ def test_frequency_is_original(self, num_cols):
         df = DataFrame(1, index=index, columns=range(num_cols))
         df.apply(lambda x: x)
         assert index.freq == original.freq
+
+    def test_func_returns_object(self):
+        df = pd.DataFrame({"a": [1, 2]}, index=pd.Int64Index([1, 2]))
```

Comment:
* nit: Use `DataFrame` instead of `pd.DataFrame`
* Reference issue number as a comment above